@xen-orchestra/web/src/components/hosts/panel/card/HostInfoCard.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <UiCard class="card-container">
+    <UiCardTitle>
+      <UiLink :to="`/host/${host.id}/dashboard`" size="medium" :icon="`object:host:${getHostState(host.power_state)}`">
+        {{ host.name_label }}
+      </UiLink>
+    </UiCardTitle>
+    <div class="content">
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('state') }}</template>
+        <template #value>
+          <span class="power-state">
+            <VtsIcon :name="powerState.icon" size="medium" />
+            {{ powerState.text }}
+          </span>
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('uuid') }}</template>
+        <template #value>{{ host.id }}</template>
+        <template #addons>
+          <VtsCopyButton :value="host.id" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue wrap>
+        <template #key>{{ t('description') }}</template>
+        <template #value>{{ host.name_description }}</template>
+        <template v-if="host.name_description" #addons>
+          <VtsCopyButton :value="host.name_description" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('tags') }}</template>
+        <template #value>
+          <UiTagsList v-if="host.tags.length > 0">
+            <UiTag v-for="tag in host.tags" :key="tag" accent="info" variant="secondary">
+              {{ tag }}
+            </UiTag>
+          </UiTagsList>
+        </template>
+        <template v-if="host.tags.length > 0" #addons>
+          <VtsCopyButton :value="host.tags.join(', ')" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('status') }}</template>
+        <template #value>
+          <VtsEnabledState :enabled="host.enabled" />
+        </template>
+      </VtsCardRowKeyValue>
+      <VtsCardRowKeyValue>
+        <template #key>{{ t('pool') }}</template>
+        <template #value>
+          <div v-if="pool" class="pool-name">
+            <UiLink :to="`/pool/${pool.id}/dashboard`" icon="object:pool" size="medium">
+              {{ pool.name_label }}
+            </UiLink>
+          </div>
```

Comment:
```suggestion
          <div v-if="pool" class="pool-name">
             <VtsIcon name="fa:city" size="medium" />
            <UiLink :to="`/pool/${pool.id}`" size="small">
              {{ pool.name_label }}
            </UiLink>
          </div>
```

you can keep this for now